tests/testthat/helper-path.R

```diff
@@ -0,0 +1,2 @@
+code_path <- dirname(dirname(getwd()))
```

Comment:
This is so janky. It's as if `testthat` doesn't want anyone to invoke non-package files 😬 

(these tests did help me, though!)

Reply:
this is not great!  i probably found the same stackoverflow post you did, indicating there is not a better answer.

Reply:
This makes my skin crawl. 

Wouldn't a better solution for this be to include a line in the `data-raw/` script that copies itself to `tests/testthat/testdata`?